api/predict.py

```diff
@@ -73,9 +73,18 @@ def post(self):
         result = {'status': 'error'}
 
         input_json = MAX_API.payload
-        for p in input_json["paragraphs"]:
-            if p["context"] == "":
-                abort(400, "Empty input, please provide a paragraph.")
+        try:
+            for p in input_json["paragraphs"]:
+                if p["context"] == "":
+                    abort(400, "Invalid input, please provide a paragraph.")
+                if not isinstance(p["questions"], list):
```

Comment:
Does testing whether this is a `Sequence` sufficient?

Reply:
I later realized this check doesn't matter anyway because flask automatically catches those errors based on our definition of the input structure.

Reply:
So since we defined `questions` as a `fields.List()`, it automatically performs this check.